<launch>
  <node pkg="uwb_localizer" type="uwb_localizer.py" name="uwb_localizer" output="screen">
    <param name="port" value="/dev/ttyUSB0"/>
    <param name="baud" value="460800/>
    <param name="frame_id" value="map"/>
    <rosparam>
      anchors:
        "1": [0.0, 0.0, 0.0]
        "3": [3.0, 0.0, 0.0]
        "7": [0.0, 3.0, 0.0]
    </rosparam>
    <param name="calibration" value="linear"/>
    <rosparam>
      calib_params: [0.001, 0.0]
    </rosparam>    
    <param name="z_sign" value="1"/>
  </node>
</launch>
